src/controls/qml/LabeledSwitch.qml

```diff
@@ -21,26 +22,45 @@ import QtQuick 2.9
 import org.asteroid.controls 1.0
 
 Row {
-    // labelWidthRatio is the ratio of label width to the total width
-    property real labelWidthRatio: 0.7143
-    // fontToHeightRatio is the ratio of the font size to the height
-    property real fontToHeightRatio: 0.3
+
     property alias checked: toggle.checked
     property alias text: label.text
 
-    Label {
-        id: label
-        text: value
-        font.pixelSize: parent.height * fontToHeightRatio
-        verticalAlignment: Text.AlignVCenter
-        wrapMode: Text.Wrap
-        width: parent.width * labelWidthRatio
-        height: parent.height
-    }
+    Item {
+        id: positioner
```

Comment:
We should either rely on the Row's positioning or replace the Row with an Item like this "positioner". Same for the other files

Reply:
Indeed, i now learned that row can have its dedicated margin left and right called rigthPadding and leftPadding. Will try to use those to make the positioner obsolete.

Reply:
The row padding limited the highlight bar width. So i converted the row into an item.